Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -1399,6 +1399,12 @@ def get_e2e_packs(self) -> set[str]:
         return {"TAXIIServer", "EDL", "QRadar", "Slack"}
 
 
+class SDKNightlyTestCollector(TestCollector):
+
+    def _collect(self) -> CollectionResult | None:
+        return CollectionResult.union((self.sanity_tests))  # type: ignore
```

Comment:
```suggestion
        return CollectionResult.union((self.sanity_tests))  # type: ignore[<kind_of_error>]
```

can we avoid it type: ignore? and if not at least state which type of mypy rule should be ignored

Reply:
yes